e2e/pages/app/toolbar-search/toolbar-search.testpage.component.html

```diff
@@ -1,83 +1,96 @@
 <div class="demo-toolbar">
 
-    <div class="demo-toolbar-left">
-        <ux-toolbar-search id="searchLeft"
-            [alwaysExpanded]="alwaysExpanded"
-            [(expanded)]="expanded"
-            direction="right"
-            background="white"
-            (search)="onSearch($event)">
-
-            <input uxToolbarSearchField
-                #searchField
-                type="text"
-                class="form-control"
-                placeholder="Search"
-                aria-label="Search"
-                [class.ux-toolbar-search-clear-offset]="searchText"
-                [(ngModel)]="searchText">
-
-            <button uxToolbarSearchButton
-                type="button"
-                class="btn btn-link btn-icon button-secondary"
-                aria-label="Toggle Search">
-                <ux-icon name="search"></ux-icon>
+        <div class="demo-toolbar-left">
+            <ux-toolbar-search id="searchLeft"
+                [alwaysExpanded]="alwaysExpanded"
+                [(expanded)]="expanded"
+                direction="right"
+                background="white"
+                (search)="onSearch($event)">
+
+                <input
+                    uxToolbarSearchField
+                    #mastheadSearchField
+                    type="text"
+                    class="form-control"
+                    placeholder="Search"
+                    aria-label="Search"
+                    [(ngModel)]="mastheadSearchText"
+                    [class.ux-toolbar-search-clear-offset]="mastheadSearchText"
+                    (keydown.escape)="searchDropdownOpen = false"
+                    (blur)="searchDropdownOpen = false"
+                    (click)="searchDropdownOpen = true"/>
+
+                <ux-typeahead
+                    #searchTypeahead
+                    class="typeahead-example"
+                    [(open)]="searchDropdownOpen"
+                    [filter]="mastheadSearchText"
+                    [options]="options"
+                    (optionSelected)="input = $event.option">
```

Comment:
I think this should be:

```html
(optionSelected)="mastheadSearchText = $event.option"
```
You can then remove the `input` property from the class